scripts/utilities/pkg_build_deb.sh

```diff
@@ -103,7 +103,14 @@ if [[ "${cross_build}" != "yes" ]]; then
 	if (( ${at_version} >= 9 )) || [[ "${at_version}" == next ]]; then
 		pushd ${deb_d} > /dev/null
 		cp /usr/bin/dh_strip .
-		patch -p1 < ${at_supported_distros}_dh_change_dest.patch
+
+		# For some distros, make sure we also use 'strip' and 'objcopy' from
+		# the binutils we just built instead of the ones from the system. This
+		# specific change will only be applied to distros with a reference to
+		# __AT_BIN__ in their *dh_change_dest.patch file.
+		patch -p1 < <(sed -e "s/__AT_BIN__/${at_dest//\//\\/}\/bin\//g" \
+				${at_supported_distros}_dh_change_dest.patch)
```

Comment:
I'll profess my ignorance... :-)  Can you explain what this does:
```
sed -e "s/__AT_BIN__/${at_dest//\//\\/}\/bin\//g"
```
The slash-slash-backslash-slash-slash-backslash-backslash-slash within `${}`, specifically.

Reply:
@ThinkOpenly This pattern was used somewhere else in the file, actually. I admit it's hard to understand. `//` replaces any ocurrences of a substring in a bash string. So this replaces any `/` in `$at_dest` with `\/` (back slash + forward slash), so `sed` can correctly print the `/` in the path when replacing the string in the file. Otherwise it would parse `/` as an expression separator. Summary: espace all `/` for `sed`.

So in the end, all that magic is used to replace `__AT_BIN__` in the file with `<path-to-AT-dir>/bin/`